src/node_contextify.cc

```diff
@@ -1407,6 +1407,17 @@ void Initialize(Local<Object> target,
 
   READONLY_PROPERTY(constants, "measureMemory", measure_memory);
 
+  Local<Object> loop_mode = Object::New(env->isolate());
```

Comment:
Are these constants used anywher?

Reply:
This seems to be an unrelated change. @Qard maybe something went wrong during rebase?

Reply:
Oops, I'll fix that tomorrow. 😅